lib/elixir/lib/inspect/algebra.ex

```diff
@@ -31,30 +31,32 @@ defmodule Inspect.Opts do
       options. Useful when implementing the `Inspect` protocol for nested structs
       to pass the custom options through.
 
-    * `:inspect_fun` (since v1.9.0) - a function to build algebra documents,
-      defaults to `Inspect.inspect/2`.
+    * `:inspect_fun` (since v1.9.0) - a function to build algebra documents.
+      Defaults to `Inspect.inspect/2`.
 
     * `:limit` - limits the number of items that are inspected for tuples,
-      bitstrings, maps, lists and any other collection of items. It does not
-      apply to printable strings nor printable charlists and defaults to 50.
+      bitstrings, maps, lists and any other collection of items. It neither
+      applies to printable strings nor to printable charlists.
```

Comment:
Is the previous sentence incorrect? I am not a native speaker, so I always struggle with neither. I am wondering if there is a clearer way to write this?

Reply:
nor is usually used after neither.

This one reads better IMO.

* `:limit` - limits the number of items that are inspected for tuples, 
bitstrings, maps, lists and any other collection of items, with the exception of 
printable strings and printable charlists.

Reply:
I love it. :+1: I would even add:

> with the exception of printable strings and printable charlists which use the `:printable_limit` option

WDYT?

Reply:
Yeah.. it looks great. pushed!